.vscode/extensions.json

```diff
@@ -2,7 +2,7 @@
   "recommendations": [
     "dbaeumer.vscode-eslint",
     "esbenp.prettier-vscode",
-    "silvenon.mdx",
+    "unifiedjs.vscode-mdx",
```

Comment:
intentional?

Reply:
it's in the PR description - [silvenon is deprecated](https://github.com/silvenon/vscode-mdx)